e2e/app/component/new-notebook-modal.ts

```diff
@@ -41,9 +41,11 @@ export default class NewNotebookModal extends Modal {
   async fillInModal(notebookName: string, language: Language): Promise<void> {
     await this.name().then( (textbox) => textbox.type(notebookName));
     if (language === Language.Python) {
-      await this.python3Radiobutton().then( (radio) => radio.select());
+      const radio = this.python3Radiobutton();
+      await radio.select();
     } else {
-      await this.RRadiobutton().then( (radio) => radio.select());
+      const radio = this.RRadiobutton();
+      await radio.select();
```

Comment:
It's a good idea. So far, I only noticed two modals with radiobuttons. A common function doesn't save a lot at this time. Also, I didn't combine this time because Python radiobutton labels were different in two modals. One is `Python 3` and other is `Python`.